pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -477,8 +482,11 @@ public int getMajorVersion()
         return MAJORVERSION;
     }
 
-
-    public static final int MINORVERSION = @MINORVERSION@;
+    public static final int MINORVERSION =
```

Comment:
the java comment preprocessor also allows such construction
public static final int MINORVERSION =/*$mvn.project.property.parsedversion.minorversion+";"$*//*-*/4;